lib/build-cron.js

```diff
@@ -22,8 +23,11 @@ module.exports = function buildCronJobs(cronConfigsDir, collectionName, logClean
 
   if (!cronConfigsDir || !collectionName) {
     const err = new VError('Error: \'cronConfigsDir\' and \'collectionName\' are required params.');
-    raygun.send(err);
-    throw err;
+    return Promise.resolve(raygun.send(err))
```

Comment:
I've tested it as much as I can without actually having the changed merged. The plan is: After this PR is merged, tag it and create a new release, then create a PR to update the version in one of the SS services, and let the cron jobs run overnight in the dev environment